package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ubiquibot/permit-generation",
-  "version": "1.2.2",
+  "version": "1.2.3",
```

Comment:
This should be taken care of by `release-please` that should actually bump it to `1.3.0` since you committed it with `feat`. To bump it to `2.0.0` you should commit with `feat!`

Reply:
So you mean it's better to keep it `1.2.2`, right?

Reply:
Yes I believe so. Should happen by merging these changes inside the main branch.

Reply:
fixed https://github.com/ubiquibot/permit-generation/pull/17/commits/74ac030203f56d341919c359c1ac83360ad18050